[2021-03-02 07:21:33,099] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-03-02T10:18:24.224590+00:00 [queued]>
[2021-03-02 07:21:33,103] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-03-02T10:18:24.224590+00:00 [queued]>
[2021-03-02 07:21:33,103] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-02 07:21:33,103] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-02 07:21:33,103] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-02 07:21:33,194] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): also_run_this> on 2021-03-02T10:18:24.224590+00:00
[2021-03-02 07:21:33,195] {standard_task_runner.py:52} INFO - Started process 31800 to run task
[2021-03-02 07:21:33,197] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', '2021-03-02T10:18:24.224590+00:00', '--job-id', '21', '--pool', 'default_pool', '--raw', '--subdir', '/media/thiagola92/LinuxHD/Documents/public/learning-apache-airflow/venv/lib/python3.7/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp8w0ljsud', '--error-file', '/tmp/tmp5a8z936v']
[2021-03-02 07:21:33,198] {standard_task_runner.py:77} INFO - Job 21: Subtask also_run_this
[2021-03-02 07:21:33,323] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.also_run_this 2021-03-02T10:18:24.224590+00:00 [running]> on host thiagola92-ubuntu
[2021-03-02 07:21:33,462] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-03-02T10:18:24.224590+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-02T10:18:24.224590+00:00
[2021-03-02 07:21:33,462] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-02 07:21:33,463] {bash.py:158} INFO - Running command: echo "run_id=manual__2021-03-02T10:18:24.224590+00:00 | dag_run=<DagRun example_bash_operator @ 2021-03-02 10:18:24.224590+00:00: manual__2021-03-02T10:18:24.224590+00:00, externally triggered: True>"
[2021-03-02 07:21:33,467] {bash.py:169} INFO - Output:
[2021-03-02 07:21:33,468] {bash.py:173} INFO - run_id=manual__2021-03-02T10:18:24.224590+00:00 | dag_run=<DagRun example_bash_operator @ 2021-03-02 10:18:24.224590+00:00: manual__2021-03-02T10:18:24.224590+00:00, externally triggered: True>
[2021-03-02 07:21:33,469] {bash.py:177} INFO - Command exited with return code 0
[2021-03-02 07:21:33,603] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20210302T101824, start_date=20210302T102133, end_date=20210302T102133
[2021-03-02 07:21:33,735] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-02 07:21:33,771] {local_task_job.py:146} INFO - Task exited with return code 0
